tycho-core/src/main/java/org/eclipse/tycho/osgi/configuration/MavenDependenciesResolverConfigurer.java

```diff
@@ -188,7 +189,14 @@ public String getName() {
 
             @Override
             public String getGroupId() {
-                return model.getGroupId();
+                String groupId = model.getGroupId();
+                if (groupId == null || groupId.isBlank()) {
+                    Parent parent = model.getParent();
+                    if (parent != null) {
+                        return parent.getGroupId();
```

Comment:
You can recurse until root is found.

Reply:
I think there is no root, as one has at least give the version of the parent in a `pom.xml` isn't it?
Or can a `<parent>` inherit from its parent the version? I think maven does not allow this.

Reply:
Yes, you're right, please ignore my dumb comment.